api/specs.proto

```diff
@@ -258,6 +258,12 @@ message ContainerSpec {
 	// task will exit and a new task will be rescheduled elsewhere. A container
 	// is considered unhealthy after `Retries` number of consecutive failures.
 	HealthConfig healthcheck = 16;
+
+	message SecurityConfig {
+	    string userns = 1;
+	    string credential_spec = 2 [(gogoproto.customname) = "CredentialSpec"];
```

Comment:
nit: `customname` is not required, I believe that's how it's going to get generated anyway.

We use custom names for stuff like `DNSConfig` (otherwise it would get generated as `DnsConfig`)